### PR TITLE
Add newsletter July

### PR DESCRIPTION
**Pull request by wipfli**
Thanks to everyone who helped writing this newsletter!

### CODE REVIEW

File: content/news/2024-07-31-maplibre-newsletter-july-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+We managed to create a WebAssembly build of MapLibre Native with a [functional demo](https://maplibre-native-wasm-dist.pages.dev/qt-opengl2/), taking advantage of the fact that it is a platform supported by Qt (Note that [Qt for WebAssembly](https://doc.qt.io/qt-6/wasm.html), unlike MapLibre Native, is not available under a permissive license). More info can be found in [this issue](https://github.com/maplibre/maplibre-native-qt/issues/49#issuecomment-2253599308). There is also an issue for making a [WebAssembly build that does not depend on Qt](https://github.com/maplibre/maplibre-native/issues/2554), but this would require implementing our own platform code for WebAssembly (for things like HTTP requests and an event loop) and a OpenGL ES to WebGL translation layer (or WebGPU backend). So this would be quite an undertaking.
```
Comment (birkskyum) [inline content/news/2024-07-31-maplibre-newsletter-july-2024/index.md:41]: There are several ways we that can move things forward from here, so maybe let's keep the tone optimistic and keep focus on what we've achieved.

Leveraging Qt for WebAssembly, we managed to create a WebGL1 build of MapLibre Native with a [functional demo](https://maplibre-native-wasm-dist.pages.dev/qt-opengl2/). More info on this demo can found [here](https://github.com/maplibre/maplibre-native-qt/issues/49#issuecomment-2253599308). This is an early stage, and we're working in multiple directions (overview [here](https://github.com/birkskyum/maplibre-native-wasm)), so better iterations and builds that doesn't depend on Qt (it's used for HTTP requests and an event loop), will follow. Note that Qt for WebAssembly has a more strict license than MapLibre Native.

Comment (louwers) [reply content/news/2024-07-31-maplibre-newsletter-july-2024/index.md:41]: Sounds good. You have a disclaimer about Qt's license in your README.

Comment (louwers) [inline content/news/2024-07-31-maplibre-newsletter-july-2024/index.md:41]:
```suggestion
Leveraging Qt for WebAssembly, we've managed to create a WebGL1 build of MapLibre Native with a [functional demo](https://maplibre-native-wasm-dist.pages.dev/qt-opengl2/). More info on this demo can found [here](https://github.com/maplibre/maplibre-native-qt/issues/49#issuecomment-2253599308). This is an early stage, and we're working in multiple directions (overview [here](https://github.com/birkskyum/maplibre-native-wasm)), so better iterations and builds that don't depend on Qt may follow.```

File: content/news/2024-07-31-maplibre-newsletter-july-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+The MapLibre Voting Members are the people who elect the MapLibre Governing Board. They can also vote on changes to the [MapLibre Charter](https://github.com/maplibre/maplibre/blob/main/CHARTER.md). Like every year, [we are inviting](https://github.com/maplibre/maplibre/issues/376) more people to the group of MapLibre Voting Members before the 2024 election. This is a great opportunity to look back at what happened last year and thank important contributors and donors of MapLibre.
+
+We will again use the email voting system OpaVote for the Governing Board election. Sometimes the OpaVote emails can end up in the spam folder. If you are a Voting Member, it might make sense to explicitly allow emails from noreply@opavote.com as a preparation.
+
+## MapLibre GL JS
+
+The main focus of this month was to push forward the globe view. A lot of effort went into the globe controls, gestures, and refactoring the transform class to be different between the globe and mercator view. This is the final stretch for the globe and we are aiming to release it by the end of this year.
```
Comment (louwers) [inline content/news/2024-07-31-maplibre-newsletter-july-2024/index.md:31]:
```suggestion
The main focus of this month was to push forward the globe view. A lot of effort went into the globe controls, gestures, and refactoring the transform class to be different between the globe and Mercator view. This is the final stretch for the globe and we are aiming to release it by the end of this year.
```

File: content/news/2024-07-31-maplibre-newsletter-july-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+We would like to repeat here our gratitude to Jakub for his amazing work on the globe implementation!
```
Comment (louwers) [inline content/news/2024-07-31-maplibre-newsletter-july-2024/index.md:33]:
```suggestion
We would like to repeat here our gratitude to Jakub Pelc for his amazing work on the globe implementation!
```

File: content/news/2024-07-31-maplibre-newsletter-july-2024/index.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+Leveraging Qt for WebAssembly, we've managed to create a WebGL1 build of MapLibre Native with a [functional demo](https://maplibre-native-wasm-dist.pages.dev/qt-opengl2/). More info on this demo can found [in this issue](https://github.com/maplibre/maplibre-native-qt/issues/49#issuecomment-2253599308). This is an early stage, and we're working in multiple directions, an overview of which can be found [here](https://github.com/birkskyum/maplibre-native-wasm)). Stay tuned for updates in this area, and be sure to get involved if using MapLibre Native on the web interests you.```
```
Comment (louwers) [inline content/news/2024-07-31-maplibre-newsletter-july-2024/index.md:41]:
```suggestion
Leveraging Qt for WebAssembly, we've managed to create a WebGL1 build of MapLibre Native with a [functional demo](https://maplibre-native-wasm-dist.pages.dev/qt-opengl2/). More info on this demo can found [in this issue](https://github.com/maplibre/maplibre-native-qt/issues/49#issuecomment-2253599308). This is an early stage, and we're working in multiple directions, an overview of which can be found [here](https://github.com/birkskyum/maplibre-native-wasm)). Stay tuned for updates in this area, and be sure to get involved if using MapLibre Native on the web interests you.
```

**Review by louwers — APPROVED** [review]

File: content/authors/hactar/_index.html [hunk 5]
```diff
@@ -0,0 +1,4 @@
+---
+title: Patrick W
```
Comment (louwers) [inline content/authors/hactar/_index.html:2]: I think his full name is Patrick W. Best https://www.linkedin.com/in/patrick-w-best-92938b4/

Consider using that for consistency.

**Review by louwers — CHANGES_REQUESTED** [review]
We should mention the voting member nomination process.

File: content/authors/hactar/_index.html (hunk 5 above)
Comment (wipfli) [reply content/authors/hactar/_index.html:2]: That is another Patrick. But let's ask him directly @hactar are you OK if we add your full name or do you prefer appearing as Patrick W?

Comment (hactar) [reply content/authors/hactar/_index.html:2]: Hehe, thats not me. I'm Patrick Wolowicz, you're welcome to use my full name if you would like ☺️

Comment (louwers) [reply content/authors/hactar/_index.html:2]: Haha another Patric W at another company called subzero.

**Pull request merged**